src/applications/hca/containers/IntroductionPageGated.jsx

```diff
@@ -0,0 +1,141 @@
+            VA.gov:
+          </strong>
+        </p>
+        <ul>
+          <li>
+            <a href="/sign-in-faq/#verifying-your-identity">
+              Read our identity verification FAQs
+            </a>
+          </li>
+        </ul>
+        <p>
+          Or call us at 1-877-222-VETS (
+          <a href="tel:+18772228387">1-877-222-8387</a>
+          ). If you have hearing loss, call TTY: 1-800-877-8339. We’re here
+          Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET)
+        </p>
+        <a className="usa-button-primary va-button-primary" href="/verify">
+          Verify your identity
+        </a>
+      </div>
+    }
+    isVisible
+    status="continue"
+  />
+);
+
+const LoggedOutContent = ({ route }) => (
+  <>
+    <SaveInProgressIntro
+      prefillEnabled={route.formConfig.prefillEnabled}
+      messages={route.formConfig.savedFormMessages}
+      downtime={route.formConfig.downtime}
+      pageList={route.pageList}
+      startText="Start the Health Care Application"
+    />
+    <HCASubwayMap />
+    <SaveInProgressIntro
+      buttonOnly
+      messages={route.formConfig.savedFormMessages}
+      pageList={route.pageList}
+      startText="Start the Health Care Application"
+      downtime={route.formConfig.downtime}
+    />
+    <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
+      <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018" />
+    </div>
+  </>
+);
+
+class IntroductionPageGated extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+
+  render() {
+    const {
+      showMainLoader,
+      showVerificationRequiredAlert,
+      showLOA3Content,
+      showLoggedOutContent,
+      route,
+    } = this.props;
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Apply for health care benefits" />
+        <p>Equal to VA Form 10-10EZ (Application for Health Benefits).</p>
+        {showMainLoader && <LoadingIndicator />}
+        {showVerificationRequiredAlert && <VerificationRequiredAlert />}
+        {showLoggedOutContent && <LoggedOutContent route={route} />}
+        {showLOA3Content && <HCAEnrollmentStatus route={route} />}
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => {
+  const {
+    isLoading: isEnrollmentStatusLoading,
+    hasServerError,
+    noESRRecordFound,
+  } = state.hcaEnrollmentStatus;
+  const loading = isProfileLoading(state) || isEnrollmentStatusLoading;
+  return {
+    showMainLoader: loading,
+    showVerificationRequiredAlert:
+      !loading && isLoggedIn(state) && isLOA1(state),
+    // If we can't get enrollment status for LOA3 users, treat them like a
+    // logged-out user (ie, just let them start a new application)
+    showLoggedOutContent:
+      !loading && (!isLoggedIn(state) || hasServerError || noESRRecordFound),
+    showLOA3Content:
+      isLoggedIn(state) &&
```

Comment:
You could move this logic into a selector so that it's easier to unit test. works well if `mapStateToProps` is just composing selectors that are all unit tested so that there's no logic in it that needs to be unit tested.